output-of-_settings-normal-index.txt

```diff
@@ -0,0 +1,6 @@
ested_depth":"20","parse":{"allow_unmapped_fields":"true"},"default_field":["*"],"derived_field":{"enabled":"true"}},"format":"0","history":{"uuid":"_na_"},"sort":{"missing":[],"mode":[],"field":[],"order":[]},"priority":"1","composite_index":"false","codec":"default","optimize_doc_id_lookup":{"fuzzy_set":{"enabled":"false","false_positive_probability":"0.2047"}},"check_pending_flush":{"enabled":"true"},"max_rescore_window":"10000","max_adjacency_matrix_filters":"100","analyze":{"max_token_count":"10000"},"knn.disk":{"vector":{"shard_level_rescoring_disabled":"false"}},"gc_deletes":"60s","searchable_snapshot":{"index":{"id":""},"shard_path_type":"FIXED","repository":"","snapshot_id":{"name":"","uuid":""}},"optimize_auto_generated_id":"true","max_ngram_diff":"1","knn.advanced":{"approximate_threshold":"0","filtered_exact_search_threshold":"-1"},"hidden":"false","translog":{"generation_threshold_size":"64mb","flush_threshold_size":"512mb","sync_interval":"5s","retention":{"size":"-1","age":"-1"},"durability":"REQUEST"},"auto_expand_replicas":"false","mapper":{"dynamic":"true"},"recovery":{"type":""},"requests":{"cache":{"enable":"true"}},"data_path":"","merge_on_flush":{"enabled":"true","max_full_flush_merge_wait_time":"10s","policy":"default"},"highlight":{"max_analyzed_offset":"1000000"},"routing":{"rebalance":{"enable":"all"},"allocation":{"enable":"all","total_primary_shards_per_node":"-1","total_shards_per_node":"-1"}},"search":{"concurrent":{"max_slice_count":"0"},"concurrent_segment_search":{"mode":"none","enabled":"false"},"slowlog":{"level":"TRACE","threshold":{"fetch":{"warn":"-1","trace":"-1","debug":"-1","info":"-1"},"query":{"warn":"-1","trace":"-1","debug":"-1","info":"-1"}}},"default_pipeline":"_none","idle":{"after":"30s"},"throttled":"false"},"fielddata":{"cache":"node"},"codec.compression_level":"3","default_pipeline":"_none","max_slices_per_scroll":"1024","shard":{"check_on_startup":"false"},"max_slices_per_pit":"1024","percolator":{"map_unmapped_fields_as_text":"false"},"allocation":{"max_retries":"5","existing_shards_allocator":"gateway_allocator"},"refresh_interval":"1s","indexing":{"slowlog":{"reformat":"true","threshold":{"index":{"warn":"-1","trace":"-1","debug":"-1","info":"-1"}},"source":"1000","level":"TRACE"}},"remote_store":{"translog":{"buffer_interval":"650ms","repository":"","keep_extra_gen":"100"},"enabled":"false","segment":{"repository":""}},"compound_format":"0.1","blocks":{"metadata":"false","read":"false","read_only_allow_delete":"false","read_only":"false","write":"false"},"max_result_window":"10000","knn":"false","store":{"hybrid":{"nio":{"extensions":["segments_N","write.lock","si","cfe","fnm","fdx","fdt","pos","pay","nvm","dvm","tvx","tvd","liv","dii","vem"]}},"stats_refresh_interval":"10s","type":"","fs":{"fs_lock":"native"},"preload":[]},"queries":{"cache":{"enabled":"true"}},"warmer":{"enabled":"true"},"max_shingle_diff":"3","knn.derived_source":{"enabled":"false"},"query_string":{"lenient":"false"}}}}}
```

Comment:
For an index without the `index.knn = true` setting, the merge policy remains as before: 2mb. 
`policy.floor_segment":"2097152b","`
I'm not sure why the json entry is nested differently between these 2 `_settings` responses.